BoxContentSDK/BoxContentSDKTests/BOXContentSDKTestCase.m

```diff
@@ -302,7 +303,8 @@ - (void)wipeAllKeychainEntries
                            (__bridge id)kSecClassIdentity];
     
     for (id secClass in secClases) {
-        NSMutableDictionary *query = [NSMutableDictionary dictionaryWithObjectsAndKeys:secClass, (__bridge id)kSecClass, nil];
+        NSDictionary *query = @{ (__bridge id)kSecClass: secClass,
+                                 (__bridge id)kSecAttrService: [BOXKeychainItemWrapper keychainServiceIdentifier] };
```

Comment:
Since OS X isn't sandboxed, we can't just tell it to wipe the entire keychain (we get a permission denied error if we try to). So we specify instead to delete all items with the Box keychain service identifier.